e2e/framework/testserver.go

```diff
@@ -490,12 +480,22 @@ func (t *TestServer) WaitForReady(ctx context.Context) error {
 	return err
 }
 
-func (t *TestServer) TxnTo(ctx context.Context, address web3.Address, method string) *web3.Receipt {
+func (t *TestServer) InvokeMethod(
+	ctx context.Context,
+	contractAddress types.Address,
+	method string,
+	fromAddress types.Address,
+	fromKey *ecdsa.PrivateKey,
```

Comment:
It is possible to calculate `fromAddress` from `fromKey`?

Reply:
Extracted a helper method that any module can now use 👍 

9080129